### PR TITLE
GA CSP values

### PR DESCRIPTION
**Pull request by Gregoor**
Looks like I missed some GA CSP values, I think that should be all (from testing it locally and reading some blogs about it).

### CODE REVIEW

File: libs/constants/index.js [hunk 1]
```diff
@@ -85,7 +85,12 @@ const CSP_DIRECTIVES = {
   "style-src": ["'report-sample'", "'self'", "'unsafe-inline'"],
   "object-src": ["'none'"],
   "base-uri": ["'self'"],
-  "connect-src": ["'self'"],
+  "connect-src": [
+    "'self'",
+
```
Comment (escattone) [inline libs/constants/index.js:90]:
```suggestion
```

File: libs/constants/index.js [hunk 2]
```diff
@@ -108,6 +113,9 @@ const CSP_DIRECTIVES = {
     "media.stage.mdn.mozit.cloud",
     "interactive-examples.mdn.mozilla.net",
     "wikipedia.org",
+
```
Comment (escattone) [inline libs/constants/index.js:116]:
```suggestion
```

**Review by escattone — APPROVED** [review]
Interesting! This lead me to read about Google Analytics and CSP as well. I wonder what the consequence would be to our GA data collection if we did **not** add `stats.g.doubleclick.net`, but let's go with this. Do you mind if I remove the blank lines? Thanks @Gregoor!

**Comment by Gregoor** [comment]
I kind of felt like visually grouping the analytics soruces, but looking at it again I'm noticing that the other things could also be grouped. So I can go either way on this!

**Comment by Gregoor** [comment]
@escattone I did some more grouping and added a comment where I thought the URLs did not speak for themselves, lmkwyt!

**Pull request merged**